packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/react/create-app-integration.md

```diff
@@ -0,0 +1,27 @@
+Before integrating Okta authentication to your React app, you need to obtain the OpenID Connect client ID from your Okta org by creating an app integration. You can create an app integration through the [Okta CLI](https://cli.okta.com/), the [Okta Apps API](/docs/reference/api/apps/), or through the [Admin Console](/docs/guides/quickstart/website/main/#using-the-admin-console).
+
```

Comment:
The React SPA guide I wrote leads with the Okta CLI. Can we do that here too? For example:

```
To create your Okta app integration quickly, you can use the CLI:

1. Run the command `okta login` to log in to your Okta developer account.
1. Run the command `okta apps create spa` to create the Okta application. When prompted for a name, use "Quickstart".
1. Specify the required redirect URI values:
   * **Redirect URI**: `http://localhost:3000/login/callback`.
   * **Post Logout Redirect URI(s)** select the default option, `http://localhost:3000`

After the Okta CLI completes creating your Okta application, it will write the Issuer and Client ID to the console. You will need these values to configure your SPA with Okta.
```

Reply:
We talked to the group about using CLI in our docs, but Chris was saying that since the startups have CLI commands, we don't need them in these "Sign in .." docs. Plus, I don't think the CLI sets "Interaction Code" grant, so we would have to go to the Console to do that.

Reply:
Since most of our customers use the sign-in widget, should the interaction code grant be enabled by default? What if the CLI automatically enabled it?

Reply:
My understanding is that for Identity Engine features, you need to enable Interaction Code. The SIW should be backward compatible so that it can handle Classic Engine, so I'm not sure if setting it by default in SIW is the way to go, but this is up to SIW team. As for Identity Engine, you will need the Interaction Code grant type enabled in the auth server, the app, and the SIW for everything to work.

Reply:
If the CLI automatically enables the Interaction Code, you need to tell users they will have code their app according to Interaction Code flow.